packages/lib-panoptes-js/src/panoptes.js

```diff
@@ -3,14 +3,14 @@
 const superagent = require('superagent')
 const { config } = require('./config')
 
-function handleErrors (error) {
+function handleError (error) {
```

Comment:
There's also a function called `handleError` in `resources/projects/helpers.js`, which could be confusing. I was looking at the helper function when I commented yesterday.

The helper raises an error, rather than handling it, so maybe rename that one to `raiseError`?

This function handles errors, so it should expect to receive an Error and, I think, return something that looks like an API response (an empty array or null maybe?) Reason for this is that the return value of a catch block is passed into the next then block in a promise chain. At the moment, any function chained after the catch would receive an error as the input, and I'm not sure that this is what the code is intended to do.

Reply:
The `handleError` here is just making sure the error gets logged to the console if the console is available, then passing it along. I can remove it to wholly put the responsibility of handling the superagent request errors on to the consumer of this library.

Reply:
Ok. If you want to pass the error back to the consumer, I think you want to rethrow here, after logging to the console. Otherwise, the catch block will have caught the error and the promise chain will continue as if the error has been handled.

Reply:
I'm thinking through this and I'm remembering now that there may be some cases where we don't want the error to be logged like the 404s for projects or users that don't have an avatar. I think I'll just remove it.